Bottom: 6053dcdce311455b97375d843be1c551f0ffc17d
Top:    e01ac1456d6337a42350fc1f1334c6a0b9b5fe99
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 22:47:26 +0000

Refresh of hid-step-add-copper-layer

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index 5855772..7ed0887 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1214,11 +1214,11 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
         }
 #endif
 
-    if (info.poly == NULL)
-      {
-        fprintf (stderr, "Skipping layer group %i, info.poly was NULL\n", group);
-        continue;
-      }
+      if (info.poly == NULL)
+        {
+          fprintf (stderr, "Skipping layer group %i, info.poly was NULL\n", group);
+          continue;
+        }
 
       objects = g_list_concat (objects,
         object3d_from_contours (info.poly,
